src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/repository/OffenderRepositoryTest.kt

```diff
@@ -0,0 +1,63 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.repository
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest
+import org.springframework.context.annotation.Import
+import org.springframework.data.domain.PageRequest
+import org.springframework.data.domain.Pageable
+import org.springframework.test.context.ActiveProfiles
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.config.AuditorAwareImpl
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.helper.builders.OffenderBookingDataBuilder
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.helper.builders.OffenderDataBuilder
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.helper.builders.Repository
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.jpa.Offender
+import uk.gov.justice.hmpps.kotlin.auth.HmppsAuthenticationHolder
+import uk.gov.justice.hmpps.test.kotlin.auth.WithMockAuthUser
+
+@DataJpaTest
+@ActiveProfiles("test")
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@Import(HmppsAuthenticationHolder::class, AuditorAwareImpl::class, Repository::class)
+@WithMockAuthUser
+class OffenderRepositoryTest {
```

Comment:
Feels a bit redundant this test given you have the integration test; we talked about trying to deprecate the OffenderDataBuilder etc and stick to the DSL

Reply:
ok yes , i created this first to text the repository in isolation then left it in. Can remove.